blog/en/blog/2023/07/18/weekly-report.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Biweekly Report (July 03 - July 16)"
+keywords: 
+- Apache APISIX
+- API Gateway
+- Weekly Report
+- Contributor
+description: The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/07/17/shxmVhkq_%E8%8B%B1%E6%96%87%E5%A4%B4%E5%9B%BE.png
+---
+
+> We have recently fixed and improved some features of Apache APISIX, including updates to the mock plugin which now supports adding headers, resolution of an error in the limit-count plugin when using the http variable in stream mode, fixing a cache key conflict issue in the etcd watch implementation, and the addition of a new feature to the admin API that allows forceful deletion of resources using the "force" parameter. Meanwhile, We are pleased to present Apache APISIX 3.4.0 with exciting new features and performance improvements. For more details, please read this biweekly report.
+
+<!--truncate-->
+
+## Introduction
+
+Apache APISIX grew up as a community from the first day it was open-sourced, and quickly became the most active open-source API gateway project in the world. These achievements are inseparable from the joint efforts of community partners.
+
+From 7.03 to 7.16, 21 contributors submitted 45 commits for Apache APISIX. Thank you for your contributions to Apache APISIX.
+
+We have recently fixed and improved some features, and the summary of the updates is as follows:
+
+1. The mock plugin now supports adding headers.
+
+2. Fixed an issue in the limit-count plugin where using the http variable in stream mode caused errors.
+
+3. Fixed a cache key conflict issue in the etcd watch implementation.
+
+4. Added a new feature to the admin API that allows forceful deletion of resources using the "force" parameter.
+
+"If you want to go fast, go alone. If you want to go far, go together." The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2023/07/17/ndDgARK1_%E6%89%80%E6%9C%89%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5.png)
```

Comment:
Here is a typo in the poster, please fix it.

![image](https://github.com/apache/apisix-website/assets/36651058/885ad3ed-0366-4276-8037-8c8d5a7e1a36)